packages/php-wasm/universal/src/lib/urls.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { toAbsoluteUrl } from './urls';
+
+describe('toAbsoluteUrl', () => {
+	it('should preserve full base url when relative url is provided', () => {
+		expect(
+			toAbsoluteUrl(
+				'/wp-admin/index.php',
+				new URL('http://localhost/scope:123/')
```

Comment:
I didn't see an example like that. My goal for this function was to be generic and test for all possible URL combinations not only things we know to expect from WordPress today.

What would help clarify this goal?

Reply:
Why choose such a goal, though? It requires a bit of fuzzy logic and reasoning about scopes without a clear advantage. It's not needed for the redirection handler. Where would that be needed?

Reply:
It felt like it could be useful long-term, but as you and Brandon pointed out this function is really specific to scopes.

This is why I ended up renaming it and moved it to the scopes package https://github.com/WordPress/wordpress-playground/pull/1978/files#diff-1846181c9ea179d1066838c075e2f452dfc8fc7dbad5b05345dc9f9895f5810eR162